qualang_tools/simulator_tools.py

```diff
@@ -15,35 +16,66 @@ def create_simulator_controller_connections(n_controllers, n_connections=12):
     :returns: The ControllerConnection object
```

Comment:
The new parameters are missing from the docstring.
Are there some case for keeping using the old method? If not, then shouldn't we had a depreciation warning to transition towards the "new" method?

Reply:
Good points!

Reply:
I decided just to remove the old method